group.py

```diff
@@ -1,5 +1,22 @@
-"""An example of how to represent a group of acquaintances in Python."""
+"An example of how to represent a group of acquaintances in Python."""
 
 # Your code to go here...
 
-my_group =
+my_group = {"Kyriazis":{"age":25,
+                        "job":"Msc Student",
+                        "friends":["Mathew","Michael"]
```

Comment:
Nice simple structure, what it you wanted to add a different type of relationship apart from friends?

Reply:
Nice and neat